bookwyrm/management/commands/generate_preview_images.py

```diff
@@ -56,12 +56,17 @@ def handle(self, *args, **options):
             self.stdout.write(" OK 🖼")
 
             # Books
-            books = models.Book.objects.select_subclasses().filter()
+            book_ids = [
+                values["id"]
+                for values in models.Book.objects.select_subclasses()
+                .filter()
+                .values("id")
+            ]
```

Comment:
I think you can use `.values_list("id", flat=True)` instead of a list comprehension here

Reply:
Oooh, I looked briefly for something like that but skimmed right over the `flat` option that's exactly what I wanted. Thanks!